jpl/cli/main_cli.py

```diff
@@ -0,0 +1,86 @@
+"""Jiggy Playbook Lint Command Line Interface"""
+import click
+import jpl
+
+MARK_TO_COLOR = {"PASSED": "green", "WARNING": "yellow", "FAILED": "red"}
+
+
+@click.command()
+@click.option(
+    "-v", "--verbose", required=False, count=True, help="Run `jpl` with verbosity."
+)
+@click.option(
+    "-s",
+    "--skip",
+    required=False,
+    default="true",
+    help="Skip `PASSED` rules in jpl report",
+)
```

Comment:
`@click.option('--verbose', '-v', is_flag=True, help="Print more output.")`

I think this is the closest thing to `store_true` in `argparse.  Might be a bit better than `"true"` as a string.

Reply:
`@click.option('--skip', '-s', is_flag=True, help="Skip `PASSED` rules in jpl report.")`

Reply:
‘is_flag’ will work here

Reply:
For `—verbose` We will want a count so we can increase verbosity